PokeDemo/Screens/PokemonList/Model/PokemonResult.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Pokemon.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+
+import Foundation
+
+struct PokemonResult: Decodable {
+    var count: Int
+    var next: String
+    var previous: String?
+    var results: [Pokemon]
+    
+    enum CodingKeys: String, CodingKey {
+        case count
+        case next
+        case previous
+        case results
+    }
```

Comment:
Do we need this?

Reply:
Covered in 04e0ac57f896573a4a6419d22c105cebd88f63b1